lib/connection.js

```diff
@@ -103,7 +102,12 @@ Connection.prototype.connect = function (port, host) {
       cert: self.ssl.cert,
       secureOptions: self.ssl.secureOptions,
       NPNProtocols: self.ssl.NPNProtocols
-    })
+    };
+    if( net.isIP(host) === 0 ) {
```

Comment:
Formatting for the check to pass:

```suggestion
    if (net.isIP(host) === 0) {
```

Reply:
(Also, no semicolons)